karton/classifier/classifier.py

```diff
@@ -213,9 +213,12 @@ def _get_extension(self, name: str) -> str:
         splitted = name.rsplit(".", 1)
         return splitted[-1].lower() if len(splitted) > 1 else ""
 
-    def _classify_filemagic(self, task: Task) -> Optional[Dict[str, Optional[str]]]:
+    def _classify_filemagic(self, task: Task) -> Dict[str, Optional[str]]:
         sample = task.get_resource("sample")
         content = cast(bytes, sample.content)
+        file_name = sample.name
+        if len(sample.content) == 0:
```

Comment:
```suggestion
        if not sample.content:
```

I think using the "pythonic" check might be better here - it will take care of cases where content is set to null as well